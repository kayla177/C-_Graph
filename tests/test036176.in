ENTITY ISFMXRHE name_ISFM journal
ENTITY DXQXM83R name_DXQX paper
ENTITY K10MVPGM name_K10M institution
RELATIONSHIP ISFMXRHE presented_at DXQXM83R 8.3
PRINT ISFMXRHE
HIGHEST
EXIT
