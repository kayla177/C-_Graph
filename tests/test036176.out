success
success
success
success
DXQXM83R
ISFMXRHE DXQXM83R 8.3
